static/app/types/organization.tsx

```diff
@@ -48,7 +48,8 @@ export interface Organization extends OrganizationSummary {
   allowJoinRequests: boolean;
   allowSharedIssues: boolean;
   attachmentsRole: string;
-  availableRoles: {id: string; name: string}[]; // Deprecated, use orgRoleList
+  availableRoles: {id: string; name: string}[];
+  // Deprecated, use orgRoleList
```

Comment:
The new line break here makes the comment's location misleading IMO.

Reply:
Deprecated should actually be annotated, which makes it show up in editors and also moves with the key properly, going to change it to that instead.

![Screenshot 2024-07-04 at 2 58 08 PM](https://github.com/getsentry/sentry/assets/6111995/1c115daa-22ff-4e16-a448-6b9c0ceda1ea)

![Screenshot 2024-07-04 at 2 58 13 PM](https://github.com/getsentry/sentry/assets/6111995/41d5cbcd-541d-4f68-86db-938aa6c76566)